src/_actions/checkedFilters.actions.js

```diff
@@ -0,0 +1,22 @@
+import constants from '../_constants';
+
+// constants
+const { ADD_CHECKED_FILTER } = constants;
+
+/**
+  * Add a checked filter
+  *
+  * @param {string} checkedFilter
+  * @return {object} type and payload
+  */
+
+const addCheckedFilter = checkedFilter =>
+  (dispatch) => {
```

Comment:
Only use dispatch when using thunk - it is unnecessary here. The two methods, `addCheckedFilter` and `addCheckedFilterSuccess` can be combined into one. I.e.
```
const addCheckedFilter = checkedFilter => ({
  type: ADD_CHECKED_FILTER,
  checkedFilter
});
```

Please read about [action creators](https://redux.js.org/basics/actions#action-creators) and [async action creators](https://redux.js.org/advanced/asyncactions#async-action-creators).
You can also read about [middlewares](https://redux.js.org/advanced/middleware) to understand how they work in redux